src/trim.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './decorator'
+
+class Trimmer < BaseDecorator
+  def correct_name
+    @nameable.correct_name.split(0, 9)
+  end
+end
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/199364927-11549843-8718-4a43-ba11-2fa27e58e72d.png)

- [ ] When i tried to trim the name it showed this error in the terminal. Kindly fix this error to make your application work as expected :+1:

Reply:
Hi, dear @HAMMAS-SALEEM 👍🏼 , I already solved it.